src/renderer/cursor_renderer/mod.rs

```diff
@@ -55,9 +56,10 @@ impl Default for CursorSettings {
             trail_size: 0.7,
             unfocused_outline_width: 1.0 / 8.0,
             smooth_blink: false,
-            vfx_mode: cursor_vfx::VfxMode::Disabled,
+            vfx_mode: cursor_vfx::VfxModeList::default(),
             vfx_opacity: 200.0,
             vfx_particle_lifetime: 1.2,
+            vfx_particle_highlight_lifetime: -1.,
```

Comment:
I will let you judge a suitable default for these two. The default of 1.2 for the particle lifetime is likely too high, since the meaning was changed and even more so for the highlight effects.